src/technochat/chat/chat.go

```diff
@@ -80,19 +99,32 @@ func (c *Chat) SendAll(msg WSMessage) {
 func (c *Chat) HandleChatBroadcast() {
 	for {
 		select {
-		case <-time.After(ChatAFKLifetime * time.Second):
```

Comment:
need some ttl for a new chat, otherwise some chats will never be removed
perhaps in a separate goroutine which waits either for the first user or ttl to expire

Reply:
added a chat AFKLifetime = 12 hours